src/server/search/search_family_test.cc

```diff
@@ -63,8 +63,37 @@ template <typename... Args> auto AreDocIds(Args... args) {
   return DocIds(vector<string>{args...});
 }
 
-TEST_F(SearchFamilyTest, CreateIndex) {
-  EXPECT_EQ(Run({"ft.create", "idx", "ON", "HASH", "PREFIX", "1", "prefix"}), "OK");
+TEST_F(SearchFamilyTest, CreateDropListIndex) {
+  EXPECT_EQ(Run({"ft.create", "idx-1", "ON", "HASH", "PREFIX", "1", "prefix-1"}), "OK");
+  EXPECT_EQ(Run({"ft.create", "idx-2", "ON", "JSON", "PREFIX", "1", "prefix-2"}), "OK");
+  EXPECT_EQ(Run({"ft.create", "idx-3", "ON", "JSON", "PREFIX", "1", "prefix-3"}), "OK");
+
+  EXPECT_THAT(Run({"ft._list"}).GetVec(), testing::UnorderedElementsAre("idx-1", "idx-2", "idx-3"));
+
+  EXPECT_EQ(Run({"ft.dropindex", "idx-2"}), "OK");
+  EXPECT_THAT(Run({"ft._list"}).GetVec(), testing::UnorderedElementsAre("idx-1", "idx-3"));
+
+  EXPECT_THAT(Run({"ft.dropindex", "idx-100"}), ErrArg("Unknown Index name"));
+
+  EXPECT_EQ(Run({"ft.dropindex", "idx-1"}), "OK");
+  EXPECT_EQ(Run({"ft._list"}), "idx-3");
+}
+
+TEST_F(SearchFamilyTest, InfoIndex) {
+  EXPECT_EQ(
+      Run({"ft.create", "idx-1", "ON", "HASH", "PREFIX", "1", "doc-", "SCHEMA", "name", "TEXT"}),
+      "OK");
+
+  for (size_t i = 0; i < 15; i++) {
+    Run({"hset", absl::StrCat("doc-", i), "name", absl::StrCat("Name of", i)});
+  }
+
+  auto info = Run({"ft.info", "idx-1"});
+  EXPECT_EQ(info.GetVec()[2], "fields");
+  EXPECT_EQ(info.GetVec()[3].GetVec().size(), 1);
+  EXPECT_THAT(info.GetVec()[3].GetVec()[0].GetVec(), testing::ElementsAre("name", "type", "TEXT"));
+  EXPECT_EQ(info.GetVec()[4], "num_docs");
+  EXPECT_THAT(info.GetVec()[5], IntArg(15));
```

Comment:
I think you can replace this with something along the lines of:
```cpp
EXPECT_THAT(Run({"ft.info", "idx-1"}),
            RespArray(ElementsAre(_, _, "fields", RespArray(ElementsAre(RespArray(ElementsAre("name", "type", "TEXT")))),
            "num_docs", IntArg(15)))));
```
(I'm probably off by some syntax errors, brackets, whatever, but hopefully this conveys roughly what I meant)

Reply:
Not sure its easier to read with the default formatting, but changed it

Reply:
If you want to force your own formatting, you can leave `//` at the end of the lines. Clang format then knows to align all `//` to the same column without merging the lines.